testdata/expected/docker/java-android-gradle-sample/default-group/java-android-gradle-sample/JavaArtifact.graph.json

```diff
@@ -1,1241 +0,0 @@
-{
```

Comment:
Docker version misses them because of the following:
- Docker image misses Android SDK
- Even if we'll manage to add Android SDK into Docker image, there will be another issue - we won't be able to generate files (java interfaces from aidl, R.java, BuildConfig.java) because source repo is mounted read-only.

I decided to remove expected Android test results in Docker mode. Together with sourcegraph/srclib#222 it allows to exclude running particular test in a given mode